versions.tf

```diff
@@ -2,9 +2,9 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
```

Comment:
```suggestion
  required_version = ">= 0.12.0"
```

@MartinCanovas please also remove the upper limit for TF version, we don't do it anymore

Reply:
@aknysh upper limit removed.